authentication-ldap/src/test/java/com/synopsys/integration/alert/authentication/ldap/LdapAuthenticationPerformerTest.java

```diff
@@ -73,17 +58,17 @@ void testGetAuthenticationType() {
 
     @Test
     void testAuthenticateEnabledFalse() {
-        ldapConfigModel.setEnabled(false);
-        assertDoesNotThrow(() -> ldapConfigAccessor.createConfiguration(ldapConfigModel));
+        validaLDAPConfigModel.setEnabled(false);
+        assertDoesNotThrow(() -> ldapConfigAccessor.createConfiguration(validaLDAPConfigModel));
 
         Authentication returnAuthentication = ldapAuthenticationPerformer.authenticateWithProvider(inputAuthentication);
         assertEquals(inputAuthentication.getPrincipal(), returnAuthentication.getPrincipal());
     }
 
     @Test
     void testAuthenticateProviderNotPresent() {
-        ldapConfigModel.setServerName("");
-        assertDoesNotThrow(() -> ldapConfigAccessor.createConfiguration(ldapConfigModel));
+        validaLDAPConfigModel.setServerName("");
+        assertDoesNotThrow(() -> ldapConfigAccessor.createConfiguration(validaLDAPConfigModel));
 
         LdapManager spiedLDAPManager = Mockito.spy(ldapManager);
```

Comment:
We don't use the spied ldapManager here so we cane remove the assignment of the spiedLDAPManager variable.